src/client/delphi_epidata.py

```diff
@@ -55,7 +55,10 @@ def _request(params):
     """Request and parse epidata."""
     try:
       # API call
-      return requests.get(Epidata.BASE_URL, params, headers=_HEADERS).json()
+      req = requests.get(Epidata.BASE_URL, params, headers=_HEADERS)
```

Comment:
maybe worth a comment saying that we prefer get for caching/logging, but fall back to post if get fails

Reply:
Added